server/scsynth/SC_CoreAudio.cpp

```diff
@@ -1598,6 +1606,9 @@ bool SC_CoreAudioDriver::DriverStart() {
     if (mWorld->mVerbosity >= 1) {
         scprintf("->SC_CoreAudioDriver::DriverStart\n");
     }
+
+    auto appIOProcFunc = (mSafetyClipThreshold > 0 && mSafetyClipThreshold < INFINITY) ? appIOProc<true> : appIOProc<false>;
```

Comment:
you've changed this one but not the other on line 1798. perhaps you should factor this into a member function?